package.json

```diff
@@ -10,6 +10,7 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
+    "@amcharts/amcharts4": "^4.8.9",
```

Comment:
You need to commit the updated package-lock.json after installing this

Reply:
Cheers!